Source/UI/IdentityUI.swift

```diff
@@ -462,6 +468,36 @@ extension IdentityUI {
         case let .byRoute(route, vc):
             handleRouteForUnpresentedUI(route: route, byPresentingIn: vc, client: client, completion: completion)
         case let .byLoginMethod(loginMethod, vc, localizedTeaserText, scopes):
+            if #available(iOS 13.0, *), case .sharedWebCredentials = loginMethod {
+                let fallback = { [weak self] in
+                    self?.show(input: .byLoginMethod(.password, presentingViewController: vc, localizedTeaserText: localizedTeaserText, scopes: scopes), client: client, completion: completion)
+                    self?.sharedWebCredentialsController = nil
+                }
+
+                sharedWebCredentialsController = SharedWebCredentialsController { [weak self] result in
```

Comment:
nit: we can have `sharedWebCredentialsController` captured by the closure, to it leaves until the closure it executed, but then we would have to provide a closure separately, not via the initialiser

Reply:
Yeah, but I dislike the idea of _intentionally_ using a closure to retain an object, rather than having a temporary class member.